packages/bpk-component-card/src/BpkCardWrapper.module.scss

```diff
@@ -0,0 +1,61 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index.scss';
+
+.bpk-card-wrapper {
+  @include bpk-card;
+  @include bpk-border-radius-md;
+  @include bpk-border-lg(var(--background-color), '');
+
+  &--header {
+    border-radius: $bpk-border-radius-sm() $bpk-border-radius-sm 0 0;
+    background-color: var(--background-color);
+    overflow: hidden;
+  }
+
+  &--content {
+    position: relative;
+    overflow: hidden;
+
+    &::before,
+    &::after {
+      position: absolute;
+      top: 0;
+      content: ' ';
+      z-index: 2;
+      width: bpk-spacing-lg();
+      height: bpk-spacing-lg();
+      background-color: transparent;
+      box-shadow: bpk-spacing-base() 0 0 0 var(--background-color);
```

Comment:
why is this line needed? I noticed it adds a bit of extra shadow to the card on bottom left and right corners when hovering over the card

Reply:
It due to there also need background color in the corner between header and card.
![image](https://user-images.githubusercontent.com/33623220/220283869-848c54de-f115-48c5-b58c-0e90b2749240.png)

Reply:
For extra shadow when hover the card, i guess it comes from content radius, has removed it.